dart/example/pubsub/stream.dart

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2021 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the EXAMPLES_LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:grpc/grpc.dart';
+
+import 'package:toit_api/toit/api/pubsub/subscribe.pbgrpc.dart';
+import 'package:toit_api/toit/model/pubsub/message.pb.dart';
+import 'package:uuid/uuid.dart';
+
+import 'subscription.dart';
+
+const String topic = "cloud:toit-api/example";
+
+/// Whether the given [error] is an error where the application should just retry.
+bool isRetryableGrpcError(GrpcError error) {
```

Comment:
You used a different set of retryable errors in the streamsheet code.
Please have a look here, whether mine are correct or not.
(I went through the list of errors and tried to pick the right ones).